app/views/organizations/adoptable_pets/show.html.erb

```diff
@@ -73,24 +73,18 @@
               <span class="text-uppercase text-primary fw-semibold ls-md"><%= t('.adoption_process') %></span>
               <!-- para -->
               <p class="fs-3  mb-5"><%= t('.please_read_faq_html', faq_link: link_to('FAQ', faq_index_path)) %></p>
-
             </div>
 
-            <div class=" row align-items-center ">
-              <div class="mb-6">
-                <h3 class="mb-3 fw-bold"><%= t('.please_know_that') %></h3>
-                <p class="mb-0 fs-4 ">
-                <ul><%= t('.please_know_that_items_html') %></ul>
```

Comment:
I don't see this  i18n used anywhere else after this change. Think we should remove it from en.yml?